lib/datasource/github/index.ts

```diff
@@ -216,6 +216,21 @@ export async function getPkgReleases({
     sourceUrl: 'https://github.com/' + repo,
     releases: null,
   };
+  try {
+    const url = `https://api.github.com/repos/${repo}/license`;
+    type LicenseInfo = {
+      download_url: string;
+      license: {
+        spdx_id: string;
+      };
+    };
+    const licenseInfo = (await ghGot<LicenseInfo>(url)).body;
+    dependency.licenseUrl = licenseInfo.download_url;
+    dependency.license = licenseInfo.license.spdx_id;
+  } catch (err) {
+    // istanbul ignore next
+    logger.debug({ repo, err }, 'Error retrieving license from github');
```

Comment:
Is it really an error? e.g. should GitHub return a response for every repo even if the repo hasn't specified a license? I'm wondering if it's a "license not found" case instead of "error" case.